src/build.d

```diff
@@ -1009,6 +1009,9 @@ void parseEnvironment()
         const hostDmd = env.getDefault("HOST_DC", null);
         env["HOST_DMD"] = hostDmd.length ? hostDmd : "dmd";
     }
+    else
+        // HOST_DMD may be defined in the environment
+        env["HOST_DMD"] = env.getDefault("HOST_DMD", null);
```

Comment:
I guess we could cache the value of `env.getDefault`

Reply:
Done